src/Symfony/Component/JsonEncoder/Mapping/PropertyMetadataLoaderInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonEncoder\Mapping;
+
+/**
+ * Loads properties encoding/decoding metadata for a given $className.
+ *
+ * This metadata can be used by the DataModelBuilder to create
+ * a more appropriate ObjectNode.
```

Comment:
More than what? Otherwise:
```suggestion
 * an appropriate ObjectNode.
```